Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/sword.yml

```diff
@@ -0,0 +1,32 @@
+# original PR by MagnusCrowe at https://github.com/new-frontiers-14/frontier-station-14/pull/887
+
+- type: entity
+  name: plasteel arming sword
+  parent: CaptainSabre
+  id: PlasteelArmingSword
+  description: An ancient design manufactured with modern materials and machines for a very specific target demographic.
+  components:
+  - type: Sharp
+  - type: Sprite
+    sprite: _NF/Objects/Weapons/Melee/armingsword.rsi
+    state: icon
+  - type: MeleeWeapon
+    wideAnimationRotation: -135
+    attackRate: 1.5
+    damage:
+      types:
+        Slash: 15
+    soundHit:
+        path: /Audio/Weapons/bladeslice.ogg
+  - type: Reflect
+    reflectProb: .1
+    spread: 90
+  - type: Item
+    sprite: _NF/Objects/Weapons/Melee/armingsword.rsi
+  - type: DisarmMalus
+  - type: Clothing
+    quickEquip: false
+    slots:
+    - back
+    - suitStorage
+    - belt
```

Comment:
almost all of these fields are inherited from captains sabre, remove duplicate components and fields with the same values